docs/2.9.0/docs/getting-started/manual-download.rst

```diff
@@ -1,18 +1,53 @@
 .. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Manually Installing the SDK
-***************************
+Manual Installation
+*******************
+
+The default way to install the SDK is to run::
+
+    curl https://get.daml.com | sh
+
+However, there are some contexts in which this is not the best option, such as when the machine you want to install the Daml SDK on does not have full access to the internet, or if you are not comfortable executing scripts directly from the internet.
+
+This page covers two topics:
+
+1. Installing the SDK from a downloaded tarball.
+2. Verifying the integrity of a downloaded tarball.
+
+Installing the SDK from a Tarball
+=================================
+
+Tarballs for the Open-Source Edition of the Daml SDK can be downloaded `from GitHub <https://github.com/digital-asset/daml/releases>`__; tarballs for the Enterprise Edition can be downloaded `from Artifactory <https://digitalasset.jfrog.io/ui/native/sdk-ee/>`__.
+
+SDK tarballs bear the SDK version string, which is not always identical to the associated Daml version string. When installing from a tarball, you need to know both the SDK version string and the Daml version string. For stable releases up to 2.8.0 included, they are the same.
```

Comment:
```suggestion
SDK tarballs bear the SDK version string. When installing from a tarball, you need to know both the SDK version string and the Daml version string. For stable releases up to and including 2.8.0, they are the same, but this may not always be the case in snapshots or later versions.
```